migrations/sqls/20210322162609-add-generating-to-batch-status-enum-up.sql

```diff
@@ -0,0 +1,20 @@
+ALTER table water.billing_batches ALTER column status TYPE varchar;
+ALTER table water.billing_batch_charge_version_years ALTER column status TYPE varchar;
+
+UPDATE water.billing_batches SET status = 'generating' where status='processing';
+UPDATE water.billing_batch_charge_version_years SET status = 'generating' where status='processing';
+
+DROP TYPE water.batch_status;
+
+CREATE TYPE water.batch_status AS ENUM (
+	'processing',
+	'review',
+	'ready',
+	'error',
+	'sent',
+	'empty',
+	'generating'
```

Comment:
I'm not certain if the new `generating` status is needed here - possibly the enum could have remained as it was?

Reply:
Removed migration in `81e36d2b`